### PR TITLE
Fix go err declaration in discovery snippets

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

File: src/main/resources/com/google/api/codegen/go/discovery_fragment.snip [hunk 1]
```diff
@@ -67,7 +67,7 @@ func main() {
         // {@TODO()} Handle error.
       }
     @else
-      err := {@methodPath(method)}({@callArg(method)}).Context(ctx).Do()
+      err = {@methodPath(method)}({@callArg(method)}).Context(ctx).Do()
```
Comment (broady) [inline src/main/resources/com/google/api/codegen/go/discovery_fragment.snip]: if err := ...; err != nil {
}

**Comment by vchudnov-g** [comment]
LGTM

Thanks!

File: src/main/resources/com/google/api/codegen/go/discovery_fragment.snip (hunk 1 above)
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/go/discovery_fragment.snip]: Oh, yeah, I like this better.

Comment (broady) [inline src/main/resources/com/google/api/codegen/go/discovery_fragment.snip]: It's actually what we had before (and have published):
https://cloud-dot-devsite.googleplex.com/storage/docs/json_api/v1/bucketAccessControls/delete#examples

Not sure why that changed.

Comment (saicheems) [inline src/main/resources/com/google/api/codegen/go/discovery_fragment.snip]: I changed it to break up the line length some, but it doesn't really save that many characters.
I've put it back.

**Pull request merged**